test_common.c

```diff
@@ -51,6 +51,18 @@ int main(int argc, char **argv)
     } else
         printf("REF mechanism\n");
 
+    char *Top = word;
+    char *pool = NULL;
+
+    if (CPYmask) {
+        /* memory pool */
+        pool = malloc(poolsize);
+        if (!pool) {
+            fprintf(stderr, "error: main(), memory exhausted.\n");
```

Comment:
Change the message to "Failed to allocate memory pool."
Later, when we introduce logging facilities such as [log.c](https://github.com/rxi/log.c), it is not necessary to dump line number and function name where logged events are triggered.

Reply:
The log message is reference from file tct.c: `error: tst_insert(), memory exhausted.` .
Does the message in tct.c need modify into the same format?

Reply:
You don't have to follow the existing messages, which are generally insane.